pkg/cmd/codespace/list.go

```diff
@@ -74,6 +75,7 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 		}
 
 		tp.AddField(c.Name, nil, cs.Yellow)
+		tp.AddField(c.DisplayName, nil, cs.Blue)
```

Comment:
A new column looks great for terminal consumption, but would break scripts that are currently consuming this output. Sadly, in order to keep backwards compatibility we can only add columns at the end, but we can't insert them in the middle or rearrange them without making that opt-in.

Reply:
This would also need to be conditional based on whether we got a display name from the API.

Reply:
How about we don't add a new column and format the Name column conditionally if we have display name? Something like:

```
NAME                                                               REPOSITORY                 BRANCH               STATE      CREATED AT
josebalius-simple-server-w66g5r3g96 (some cool name)               josebalius/simple-server   master               Shutdown   Dec  2, 2021
```

I think current CLI users are used to seeing the identifier (and it's useful for URL hacking) and thus adding the display name makes sense to me.

Reply:
> How about we don't add a new column and format the Name column conditionally if we have display name? Something like:
> 
> ```
> NAME                                                               REPOSITORY                 BRANCH               STATE      CREATED AT
> josebalius-simple-server-w66g5r3g96 (some cool name)               josebalius/simple-server   master               Shutdown   Dec  2, 2021
> ```
> 
> I think current CLI users are used to seeing the identifier (and it's useful for URL hacking) and thus adding the display name makes sense to me.

I like this.

Reply:
I modified it like @josebalius suggested